Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.13 as default
+FROM alpine:3.15.0 as default
```

Comment:
A few months ago we converted all our team's docker images to use `FROM alpine:3` so they'd always use the latest stable release (prompted by this same issue). I'd suggest copying that pattern unless there is a reason not to... and if there is I'd be curious to know so as to evaluate whether it applies to the other projects. Thanks.

Reply:
Applied that change! Thank you for recommending that improvement 💯